pkg/util/flag/flags.go

```diff
@@ -47,5 +49,31 @@ func WarnWordSepNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedNam
 func InitFlags() {
 	pflag.CommandLine.SetNormalizeFunc(WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
+	ParseFlags()
+}
+
+func ParseFlags() {
```

Comment:
IMO we should not be testing flag parsing in a way that uses the test's invocation's own arguments, that seems quite broken.